views/repo_list.pug

```diff
@@ -4,6 +4,24 @@ h1
 .git-repo-areas
 
   .git-repo( ng-repeat="repo in globalState.user.repos()" ng-controller="ListItemCtrl" )
-    .repo-name
-      span( ng-bind="repo.full_name" )
-      a.link-out( ng-href="{{repo.html_url}}" target="_blank" )
+    .repo__name
+      a( ng-href="{{repo.html_url}}" ng-bind="repo.name" target="_blank" )
+      span( ng-click="getNotifications(repo.full_name)")  - Get Notifications!
+    ul.repo__notifications
+      span.close( ng-click="closeNotifications()" ng-if="notifications.length > 0" ) x
+      li( ng-repeat= "notification in notifications" )
+        span.type( ng-bind="notification.subject.type" )
+        span.title( ng-bind="notification.subject.title" )
+    .repo__description
+      span( ng-bind="repo.description" )
+    .repo__info
+      .repo__info--footer
+        - var repoType = '{{ repo.language }}'
```

Comment:
you could have added this in the ListItemCtrl instead of mixing javascript variables definition in the template files...